canopen/pdo/base.py

```diff
@@ -313,43 +313,49 @@ def add_callback(self, callback: Callable[["Map"], None]) -> None:
         """
         self.callbacks.append(callback)
 
-    def read(self) -> None:
+    def read(self, from_od=False) -> None:
         """Read PDO configuration for this map using SDO."""
-        cob_id = self.com_record[1].raw
+
+        def _raw_from(param):
```

Comment:
Do we need an underscore name here?  The function is only defined within this scope, so there is nothing to be gained by trying to hide it?

Reply:
No, not really. Only a convention. My habit is to name functions that are defined within a fn or meth using a underscore prefix, but it is not necessity.